wasmplugin/plugin.go

```diff
@@ -487,18 +487,18 @@ func retrieveAddressInfo(target string) (string, int) {
 		}
 	}
 	srcPortRaw, err := proxywasm.GetProperty([]string{target, "port"})
-	if err != nil {
+	if err == nil {
+		targetPort, err = parsePort(srcPortRaw)
```

Comment:
This is srcPortRaw, not target.

Anyways let's add a helper that accepts an attribute name and returns a port